README.md

```diff
@@ -4,8 +4,46 @@
 
 > Feel free to explore my work
 
+![screenshot](https://drive.google.com/file/d/1Biwj7V5NOQTpdYbSaQnY2ze8rRn8ODrP/view?usp=sharing)
+
 ## Built With
 
 - HTML
 - CSS
-- JS
\ No newline at end of file
+- JS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+
+N/A
```

Comment:
- [ ] It could be a good idea to provide instructions on how to set up the project locally, so that other people may have an idea on how to do so 😉